.changeset/quick-owls-chew.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I would not do releases for README changes that are not critical like completely wrong statements or omissions that prevent being able to use it at all. They can go into the next release with other stuff.   ->  delete that file.